src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String task;
+    protected Boolean isDone;
+
+    public Task(String task) {
+        this.task = task;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]"); // mark done task with X
+    }
+
+    public Boolean getStatus() {
```

Comment:
Consider renaming this boolean method to sound like booleans.
```suggestion
    public Boolean isDone() {
```